Feature/loguru

shatteredbeam opened the pull request:
Removed all instances of LOG and custom logging to switch over to using loguru.

shatteredbeam committed:
unfinished, first commit

shatteredbeam committed:
pipfile updated for loguru

shatteredbeam committed:
Remove all instances of LOG, replace logging system with loguru.

shatteredbeam committed:
pycodestyle fixes

shatteredbeam committed:
more pycodestyle fixes :/

theunkn0wn1 commented on src/config/__init__.py:
```diff
@@ -78,4 +83,20 @@
+
+# Hook logging intercept
+logging.basicConfig(handlers=[InterceptHandler()], level=0)
+logger.info("Logging Hook Executed.  As a workaround, log messages sent to logging will be "
```
[question] do we really still need this intercept handler? I thought by migrating everything to logaru we wouldn't need to do this?

shatteredbeam replied:
Yep.  It'll pull logging messages from other input sources (ahem, Pydle) and push them into our instance.

theunkn0wn1 reviewed (dismissed):
LGTM, will do one final review (by running the code) tomorrow at the latest 😇

shatteredbeam committed:
Remove logging hook message, I already converted everything.

Log value separation done by __name__ space instead of by module, which is more friendly to read.

theunkn0wn1 reviewed (approved):
👍 LGTM
